packages/web-twig/src/Resources/components/TextField/README.md

```diff
@@ -56,6 +56,52 @@ Without lexer:
 }} %}
 ```
 
+## Input Width
+
+There are several ways to adjust the input width:
+
+### `inputWidth` Attribute
+
+The `inputWidth` attribute is supported on inputs of the following types: `email`,
+`password`, `tel`, `text`, `url`.
+
+```twig
+<TextField
+  id="textFieldWidthCh"
+  inputWidth="4"
+  label="Input width wide 4 characters"
+  name="textFieldWidthCh"
+  placeholder="Placeholder"
+  type="text"
+/>
+```
+
+This option is generally recommended for inputs with a limited value length
+(e.g. numeric representation of day, month, year). Supported values are `2`, `3`
+and `4` (characters).
+
+### `--width` CSS Custom Property
+
+If you need any other value or prefer using `em` unit
+instead of default `ch`, define a `--width` CSS custom property on the `<input>`
+element via `inputProps` attribute:
+
+```twig
+<TextField
+  id="textFieldWidthCh"
+  inputProps="{{ { style: '--width: 10ch' } }}"
```

Comment:
```suggestion
  inputProps="{{ { style: '--width: 10ch' } }}"
  inputWidth="10"
```

Reply:
We have to set invalid value for `inputWidth otherwise style via CSS property isn't applied. Not intuitive.